swift/extractor/translators/DeclTranslator.cpp

```diff
@@ -401,4 +401,23 @@ std::optional<codeql::OpaqueTypeDecl> DeclTranslator::translateOpaqueTypeDecl(
   return std::nullopt;
 }
 
+static int translateDiagnosticsKind(swift::DiagnosticKind kind) {
```

Comment:
I guess we should find a way to combine this with the other instance https://github.com/github/codeql/pull/11101/files#diff-05ba0db58312ad56f391863b26935ca619b3808897ed517a4cc68b2ea97be111R13

Reply:
yes, I was considering it, but we can do it with a later PR if you prefer :slightly_smiling_face:

Reply:
ah, no wait, your PR is merged, so I can do it :slightly_smiling_face: